src/elements/content-sidebar/Sidebar.js

```diff
@@ -117,6 +129,36 @@ class Sidebar extends React.Component<Props, State> {
         history.push(`${history.location.pathname}/versions`);
     };
 
+    /**
+     * Getter/setter for sidebar forced state
+     *
+     * @param isOpen - Optionally set the sidebar to open/closed
+     * @returns {boolean} - The sidebar open/closed state
+     */
+    isForced(isOpen?: boolean): ?boolean {
+        if (isOpen !== undefined) {
+            this.store.setItem(SIDEBAR_FORCE_KEY, isOpen ? SIDEBAR_FORCE_VALUE_OPEN : SIDEBAR_FORCE_VALUE_CLOSED);
+        }
+
+        return this.store.getItem(SIDEBAR_FORCE_KEY);
```

Comment:
What if we fell back to`false` if the key doesn't exist? Could clean up our types a bit.

Reply:
I also use this method to determine if the value is set at all or is null. I'll update the flow types, though.